### PR TITLE
Merge of several simple PR's for the next tag

### PR DESCRIPTION
**Pull request by ekluzek**
### Description of changes

Merge of several simple PR's for the next tag. Some from John Alex with history updates and improved documentation.
Some from Sam Rabin for crop calendar testing. One from Greg Lemieux to add xesmf to the conda environment.
A simple NEON update. And a few simple fixes.

### Specific notes

Contributors other than yourself, if any: @johnpaulalex @samsrabin @glemieux 

CTSM Issues Fixed (include github issue #):

Are answers expected to change (and if so in what way)?

Any User Interface Changes (namelist or namelist defaults changes)?

Testing performed, if any: Regular testing will be done
Each PR was tested on it's own.

### CODE REVIEW

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -10,5 +10,8 @@ b88e1cd1b28e3609684c79a2ec0e88f26cfc362b
 b771971e3299c4fa56534b93421f7a2b9c7282fd
 9de88bb57ea9855da408cbec1dc8acb9079eda47
 8bc4688e52ea23ef688e283698f70a44388373eb
+0a5a9e803b56ec1bbd6232eff1c99dbbeef25eb7
+810cb346f05ac1aabfff931ab1a2b7b584add241
+5933b0018f8e29413e30dda9b906370d147bad45
 # Ran SystemTests and python/ctsm through black python formatter
 5364ad66eaceb55dde2d3d598fe4ce37ac83a93c
```
Comment (slevis-lmwg) [inline .git-blame-ignore-revs:17]: I recognize most of this PR's mods as ones that I reviewed in #2081, which got merged in 2023. Typing
`git diff samsrabin/make-fsurdat-ace-dev ekluzek/simplemerge`
reveals a few diffs. In this file here, Erik's PR removes a line that we should probably keep.

File: cime_config/SystemTests/rxcropmaturity.py [hunk 2]
```diff
@@ -255,19 +255,28 @@ def _run_make_fsurdat_all_crops_everywhere(self):
             raise RuntimeError(error_message)
 
         # Where we will save the fsurdat version for this test
-        self._fsurdat_out = os.path.join(self._path_gddgen, "fsurdat.nc")
+        path, ext = os.path.splitext(self._fsurdat_in)
+        dir_in, filename_in_noext = os.path.split(path)
+        self._fsurdat_out = os.path.join(self._path_gddgen, f"{filename_in_noext}.all_crops_everywhere{ext}")
```
Comment (slevis-lmwg) [inline cime_config/SystemTests/rxcropmaturity.py:260]: In this line Erik's PR adds _path_gddgen. I will let @ekluzek and @samsrabin decide if Erik's is the correct version.

**Review by slevis-lmwg — CHANGES_REQUESTED** [review]
- [ ] Erik's PR needs an update from dev133 to the latest tag. I suspect that conflicts will arise that @samsrabin is likely best suited to resolve.
- [ ] I recommend looking at the output of 
`git diff samsrabin/make-fsurdat-ace-dev ekluzek/simplemerge`
to confirm my assertion that most of Erik's updates to #2081 are minor and fine. I called out only two of those updates that I want @samsrabin and @ekluzek to please evaluate more carefully.
- [ ] Once this PR has the latest tag merged with it, please run the tests. I will let @ekluzek and @samsrabin decide which tests are required.

File: .git-blame-ignore-revs (hunk 1 above)
Comment (samsrabin) [reply .git-blame-ignore-revs:17]: Yes, `540b256d1f3382f4619d7b0877c32d54ce5c40b6` should be added after line 17.

Comment (slevis-lmwg) [reply .git-blame-ignore-revs:17]: Note that it's already present in main, so updating to the latest tag will bring it in, as long as conflicts are resolved correctly.

File: cime_config/SystemTests/rxcropmaturity.py (hunk 2 above)
Comment (samsrabin) [reply cime_config/SystemTests/rxcropmaturity.py:260]: I think the only difference is that my version satisfies black/pylint.

**Comment by samsrabin** [comment]
@ekluzek I think the best thing to do would be for you to close this PR. Then you would set the other PRs (the ones that aren't already merged, at least—so not #2081) to merge to b4b-dev.

File: cime_config/SystemTests/rxcropmaturity.py (hunk 2 above)
Comment (slevis-lmwg) [reply cime_config/SystemTests/rxcropmaturity.py:260]: Oh good, you're right.

**Comment by slevis-lmwg** [comment]
@samsrabin I agree because the non-2081 updates are tiny from what I can tell.

**Comment by wwieder** [comment]
@ekluzek will look at this and move remaining items to b4b-dev.

**Comment by wwieder** [comment]
This seems to have fallen off the radar.  Does this PR need to be merged?  On what timeline?

**Comment by samsrabin** [comment]
I'm pretty sure all this has been merge already; a test merge into master revealed no changed files.

**Comment by wwieder** [comment]
@samsrabin looked at this and found they were all merged already

**Pull request closed without merging**